helix-lock/src/main/java/org/apache/helix/lock/helix/ZKLockConfig.java

```diff
@@ -0,0 +1,173 @@
+package org.apache.helix.lock.helix;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.helix.lock.LockScope;
+import org.apache.helix.manager.zk.GenericZkHelixApiBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Hold configs used for a ZK distributed nonblocking lock.
+ */
+public class ZKLockConfig {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(org.apache.helix.lock.helix.ZKLockConfig.class.getName());
+  private LockScope _lockScope;
+  private String _zkAddress;
```

Comment:
I do have some concerns about this direct ZkAddress parameter. This won't support multi-zk usage.
It might be OK, but the LOCK API will be treated as a special case, I guess.

Reply:
Please see the above comment. I feel multi ZK should be implicitly supported. Let me know if that's not the case.